operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -165,6 +165,8 @@ type CassandraDatacenterSpec struct {
 
 	// Cassandra users to bootstrap
 	Users []CassandraUser `json:"users,omitempty"`
+
+	StaticSeedIPs []string `json:"staticSeedIPs,omitempty"`
```

Comment:
we should think hard about this name... @bradfordcp used the word "hint" 🤔 

https://github.com/datastax/cass-operator/issues/131

Reply:
Do we want to specify seed entries at the top-level of the CRD or in something like `config.cluster-info.seeds` per https://github.com/datastax/cass-config-builder/blob/master/README.md#cluster-info-key?

Reply:
What happens if a user specifies both?

Reply:
I don't think we should be putting this stuff under `config` btw, and if they specify both we overwrite the config-blob one